Source/ThirdParty/ANGLE/ANGLE.xcodeproj/project.pbxproj

```diff
@@ -3985,6 +5246,753 @@
 			};
 			name = Production;
 		};
+		7BFBD7AF2C7DE55300BFCA94 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 7BFBD7E02C7DE7C700BFCA94 /* ANGLETranslatorFuzzerStaticAssert.xcconfig */;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
+				ASSETCATALOG_COMPILER_GENERATE_SWIFT_ASSET_SYMBOL_EXTENSIONS = YES;
+				CLANG_ANALYZER_NONNULL = YES;
```

Comment:
You need to manually delete all of these `buildSettings` since there is a `*.xcconfig` file to use (except maybe `PRODUCT_NAME`, unless that's in the xcconfig file, too).

Probably easiest just to open the `project.pbxproj` in a text editor and delete the lines (as long as the text editor doesn't try to change tabs to spaces or make other whitespace changes without explicit action).

This is the worst part about adding new targets if you don't copy them—Xcode re-adds all these settings after folks painstakingly moved them out to xcconfig files earlier.